### PR TITLE
Docs(web-twig): Introduction TextArea stories

### PR DESCRIPTION
**Pull request by tomassychra**

### CODE REVIEW

**Review by literat — APPROVED** [review]

**Comment by literat** [comment]
Please, update snapshots.

File: packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig [hunk 1]
```diff
@@ -83,9 +83,7 @@
             {{ _placeholderAttr }}
             {{ _disabledAttr }}
             {{ _requiredAttr }}
-        >
-            {{ _value }} {# Intentionally without `raw` to prevent XSS. #}
-        </textarea>
+        >{{ _value }}</textarea>{# Intentionally without `raw` to prevent XSS. #}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig:86]: 👍

File: packages/web-twig/src/Resources/components/TextArea/stories/TextAreaValidationState.twig [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    message="Validation message"
+    name="textareaSuccess"
+    placeholder="Placeholder"
+    validationState="success"
+>
+    Filled
+</TextArea>
+<TextArea
+    id="textareaWarning"
+    isRequired={true}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/TextArea/stories/TextAreaValidationState.twig:33]:
```suggestion
    isRequired
```
useless boolean set. Please remove everywhere.

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/src/Resources/components/TextArea/stories/TextAreaValidationState.twig (hunk 2 above)
Comment (tomassychra) [reply packages/web-twig/src/Resources/components/TextArea/stories/TextAreaValidationState.twig:33]: done

**Pull request merged**